examples/with-docker/Dockerfile

```
@@ -19,8 +19,8 @@ WORKDIR /app
 
 ENV NODE_ENV production
 
-RUN addgroup -g 1001 -S nodejs
-RUN adduser -S nextjs -u 1001
+RUN addgroup --gid 1001 --system nodejs
+RUN adduser --system --uid 1001 nextjs
```

```suggestion
RUN addgroup --system --gid 1001 nodejs
RUN adduser --system --uid 1001 nextjs
```

Let's have the flags in the same order as well